app/build.gradle

```diff
@@ -54,6 +54,8 @@ dependencies {
     kapt libraries.daggerCompiler
     implementation libraries.glide
     implementation libraries.gson
+    implementation libraries.kotlinCoroutinesAndroid
```

Comment:
We won't need this dependency explicitly if/when we switch to using ViewModels and include the lifecycle viewmodel ktx dependency. The reason it's needed is because `Dispatchers.Main` for Android is included in that dependency.

https://twitter.com/ataulm/status/1261964155864711170/photo/1